test/spec/auth/auth.rst

```diff
@@ -0,0 +1,1218 @@
+.. role:: javascript(code)
```

Comment:
I'd prefer not to merge in the whole spec if possible, seems like needless text duplication across all of our driver repos